README.md

```diff
@@ -47,5 +47,16 @@ This demo should work wiht Node.js 15 (current LTS) as well as Node.js 13. See `
     docker build -t health-wallet-demo .
     docker run --rm -it --env SERVER_BASE=http://localhost:8080/api -p 8080:8080 health-wallet-demo
 
+## Run dev-only containers with watchers in Docker-Compose
+    export USER=$(id -u) 
+    docker-compose --env-file ./compose.env up
+   
+You can use the docker-compose.yaml file to spin up two dev containers with watchers, one for the UI and one for the server.
+
+### Note the following:
+1. Both containers have their `src` directories bind-mounted to the local directory's `src` folder. Any changes made in the `dev` container (or the host) will propagate to both containers + host and be registered by the watchers. This is helpful since you can, for instance, launch programs inside the `dev` container and utilize dev dependencies without needing to ever install them locally.
+2. Both containers have their `dist` folders mounted to a named volume. This means the parcel watcher in the `dev-ui` container can write changes that are accessible by the `dev` container. Note the `dev-ui` container needs `root` priviliges for this. See this [issue](https://github.com/moby/moby/issues/2259) for details. 
```

Comment:
Would it be possible to avoid this root permissions issue by mounting a `dist` directory from the host into the dev containers?

Reply:
Yeah we can get around this that way. That's probably the simplest answer. See this [commit](https://github.com/microsoft-healthcare-madison/health-wallet-demo/commit/38b456cc0a14955d280b19f754181932ef58b868) for the fix.

Reply:
Perfect. I've created a PR from you branch so you don't need to go through the trouble of re-opening.